rpmio/base64.cc

```diff
@@ -58,18 +58,21 @@ static char *base64_encode_block(const char *plaintext_in, int length_in, char *
 
 #define BASE64_DEFAULT_LINE_LENGTH 64
 
-char *rpmBase64Encode(const void *data, size_t len, int linelen)
+char *rpmBase64Encode(const void *data, size_t len, int linelen_)
```

Comment:
Ew, don't change parameter names for internal tweaks, this makes the code mightily confusing.